src/dotnet/Chat.Service/ChatsBackend.cs

```diff
@@ -63,6 +63,24 @@ public ChatsBackend(IServiceProvider services) : base(services)
         return chat with { Picture = media };
     }
 
+    // [ComputeMethod]
+    public virtual async Task<Chat?> GetTemplatedChatFor(ChatId templateId, UserId userId, CancellationToken cancellationToken)
+    {
+        if (templateId.IsNone)
+            throw new ArgumentOutOfRangeException(nameof(templateId));
+        if (userId.IsNone)
+            throw new ArgumentOutOfRangeException(nameof(userId));
+
+        var dbContext = CreateDbContext();
+        await using var _ = dbContext.ConfigureAwait(false);
+        var dbChat = await dbContext.Chats.AsQueryable()
```

Comment:
Why do you `.AsQueryable()` here?

Reply:
Don't see real reason to do so, just copied existing code. I have removed all cases with .AsQueryable()